bpfassets/libbpf/src/kepler.bpf.c

```diff
@@ -198,13 +241,27 @@ struct sched_switch_info {
 SEC("tp/sched/sched_switch")
 int kepler_sched_switch_trace(struct sched_switch_info *ctx)
 {
-	u32 prev_pid, tgid, cpu_id;
-	u64 pid_tgid, cgroup_id, cur_ts;
-	pid_t cur_pid;
+	u32 prev_pid, next_pid, cpu_id;
+	u64 *prev_tgid;
+	long prev_state;
+	u64 cur_ts = bpf_ktime_get_ns();
 
-	struct process_metrics_t *cur_pid_metrics, *prev_pid_metrics;
+	struct process_metrics_t *prev_tgid_metrics, *curr_tgid_metrics;
```

Comment:
It seems unnecessary to rename these and reorder them.
Can we pick a naming convention and stick with it - either `cur` or `curr`.
Then sorting alphabetically is always preferred.

Reply:
Yes, it should not be updated!

Reply:
Right, let's use `cur` only.